static/monorail-2.0.yaml

```diff
@@ -298,7 +298,7 @@ definitions:
     type: object
 info:
   title: RackHD 2.0
-  version: 0.0.1
+  version: 0.1.0
```

Comment:
I need to change the version number or these changes won't be published into bintray.  The bintray pdf only updates when it sees the version has changed